lang_qc/db/helper/wells.py

```diff
@@ -147,12 +147,23 @@ class PacBioPagedWellsFactory(PagedStatusResponse):
         description="A SQLAlchemy Session for the ml warehouse database",
     )
 
+    # For MySQL it's OK ot use case-sensitive comparison operators since
+    # its string comparisons are string-insensitive.
```

Comment:
subject to collation. Ours is `utf8mb4_0900_ai_ci` which is mercifully insensitive.